packages/angular/src/generators/ngrx/lib/generate-files.ts

```diff
@@ -12,22 +12,13 @@ export function generateNgrxFilesFromTemplates(
 ): void {
   const name = options.name;
   const moduleDir = dirname(options.module);
-  const templatesDir =
-    !options.syntax || options.syntax === 'creators'
-      ? './files/creator-syntax'
-      : './files/classes-syntax';
   const projectNames = names(name);
 
-  generateFiles(
-    tree,
-    joinPathFragments(__dirname, '..', templatesDir),
-    moduleDir,
-    {
-      ...options,
-      tmpl: '',
-      ...projectNames,
-    }
-  );
+  generateFiles(tree, joinPathFragments(__dirname, '..', 'files'), moduleDir, {
+    ...options,
+    tmpl: '',
+    ...projectNames,
```

Comment:
I know you didn't touch this, it looks more like a formatting thing, and this is a real nitpick of a comment (😅 ) but can we change this to
```suggestion
    ...options,
    ...projectNames,
    tmpl: '',
```

just in case, for any reason, `projectNames` _might_ include `tmpl` or in case any one comes along after and spreads a new object below `projectNames`.